sdk/samples/iot/CMakeLists.txt

```
@@ -83,6 +83,29 @@ target_link_libraries(paho_iot_hub_pnp_sample
     az::iot::samples::common
 )
 
+# PnP Component Sample
+add_executable (paho_iot_hub_pnp_component_example
```

Change `example` to `sample` like the rest?
```suggestion
add_executable (paho_iot_hub_pnp_component_sample
```

---

ahhh yes thank you
https://github.com/Azure/azure-sdk-for-c/pull/1018